aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -332,25 +317,6 @@ module aptos_framework::account {
         option::fill(&mut account_resource.rotation_capability_offer.for, recipient_address);
     }
 
-    /// Accept rotation capability from `offerer_address` if there's an existing rotation capability offer to the account owner in the offerer's account
-    public fun accept_rotation_capability_ed25519(account: &signer, offerer_address: address) : RotationCapability acquires Account {
```

Comment:
Why delete this? offer_rotation_capability_ed25519 is still there so without accept_rotation_capability_ed25519, how would people claim that offer?